src/components/OrderBreakdown/LineItemBookingPeriod.js

```diff
@@ -45,10 +55,14 @@ const BookingPeriod = props => {
             <FormattedMessage id="OrderBreakdown.bookingEnd" />
           </div>
           <div className={css.dayInfo}>
-            <FormattedDate value={endDate} {...timeFormatOptions} {...timeZoneMaybe} />
+            {(isProvider || lineItemIsEstimated || service !== SERVICE_RESCUE) ?
```

Comment:
Comparison directly would be great for cases where I only need the comparison value once. In contrast if we need to use it multiple time, saving comparison result as a constant is a better choice, since we do not need to re-compare the values every time. Beside, for some cases, using a named constant is more clear. I should have used a constant instead.